proxyd/server.go

```diff
@@ -654,6 +718,27 @@ func randStr(l int) string {
 	return hex.EncodeToString(b)
 }
 
+// timeoutLoop runs at the interval set by 'timeout' and deletes filters
+// that have not been recently used. It is started when the Server is created.
+func (s *Server) filterTimeoutLoop() {
+	ticker := time.NewTicker(s.filterTimeout)
+	defer ticker.Stop()
+	for {
+		<-ticker.C
+		s.filtersMu.Lock()
+		for id, f := range s.filters {
+			select {
+			case <-f.deadline.C:
+				// just delete, the node will automatically expire it on their end
+				delete(s.filters, id)
```

Comment:
What is the default filter timeout on the node itself? 
Is the user of this RPC able to override this? 
Adding too many filters to a node may slow down performance and not actually deleting in the filter timeout by proxyd could be a concern